packages/plugins/response-cache-redis/src/redis-cache.ts

```diff
@@ -0,0 +1,115 @@
+import Redis from 'ioredis';
+import type { Cache } from '@envelop/response-cache';
+
+export type BuildRedisEntityId = (typename: string, id: number | string) => string;
+export type BuildRedisResponseOpsKey = (responseId: string) => string;
+
+export type RedisCacheParameter = {
+  /**
+   * Redis instance
+   * @see Redis.Redis https://github.com/luin/ioredis
+   */
+  redis: Redis.Redis;
+  /**
+   * Maximum amount of items in the cache. Defaults to `Infinity`.
+   */
+  max?: number;
```

Comment:
This parameter is unused, so we can remove it

Reply:
Right, the max was helpful in the LRU in-memory-cache but not necessarily needed here. Imagine if too many keys (w/out expiration) happens in the reds store, one could flush if started to approach max storage.